### PR TITLE
LTD-3729-NEW-tag-bug

### PR DESCRIPTION
**Pull request by seijihg**
https://uktrade.atlassian.net/browse/LTD-3729

addressed full stop comment from Collin:
https://uktrade.atlassian.net/browse/LTD-3693

### CODE REVIEW

File: caseworker/templates/users/mentions.html [hunk 1]
```diff
@@ -33,7 +33,8 @@ <h1 class="govuk-heading-l">
 						<td class="govuk-table__header" scope="row">
 								{{ user_case_note_mention.reference_code }}
 						</td>
-						<td class="govuk-table__cell"> {% if user_case_note_mention.is_accessed %}<p class="govuk-tag">NEW</p>{% endif %} <br>{% if user_case_note_mention.is_urgent %}<p class="govuk-tag govuk-tag--red">URGENT</p>{% endif %} </td>
+						{{user_case_note_mention.is_accessed}}
```
Comment (kevincarrogan) [inline caseworker/templates/users/mentions.html:36]: I don't think this is meant to be here.

Comment (seijihg) [reply caseworker/templates/users/mentions.html:36]: ah right haha :P

File: caseworker/templates/users/mentions.html [hunk 2]
```diff
@@ -33,7 +33,7 @@ <h1 class="govuk-heading-l">
 						<td class="govuk-table__header" scope="row">
 								{{ user_case_note_mention.reference_code }}
 						</td>
-						<td class="govuk-table__cell"> {% if user_case_note_mention.is_accessed %}<p class="govuk-tag">NEW</p>{% endif %} <br>{% if user_case_note_mention.is_urgent %}<p class="govuk-tag govuk-tag--red">URGENT</p>{% endif %} </td>
+						<td class="govuk-table__cell"> {% if not user_case_note_mention.is_accessed %}<p class="govuk-tag">NEW</p>{% endif %} <br>{% if user_case_note_mention.is_urgent %}<p class="govuk-tag govuk-tag--red">URGENT</p>{% endif %} </td>
```
Comment (saruniitr) [inline caseworker/templates/users/mentions.html:36]: One small issue - if a case note mention is accessed then the condition won't satisfy and we end up adding `<br>` before the urgent tag, is that expected?

Comment (seijihg) [reply caseworker/templates/users/mentions.html]: yep ur right!

File: caseworker/templates/users/mentions.html [hunk 3]
```diff
@@ -27,13 +27,20 @@ <h1 class="govuk-heading-l">
 					<th class="govuk-table__header" scope="col"></th>
 				</tr>
 			</thead>
+			hahaha
```
Comment (kevincarrogan) [inline caseworker/templates/users/mentions.html:30]: You've left debug here.

File: caseworker/templates/users/mentions.html [hunk 4]
```diff
@@ -27,13 +27,20 @@ <h1 class="govuk-heading-l">
...
+			
 			<tbody class="govuk-table__body">
 				{% for user_case_note_mention in user_mentions.mentions %}
 					<tr class="govuk-table__row" id="mentions-row-{{ forloop.counter }}">
 						<td class="govuk-table__header" scope="row">
 								{{ user_case_note_mention.reference_code }}
 						</td>
-						<td class="govuk-table__cell"> {% if user_case_note_mention.is_accessed %}<p class="govuk-tag">NEW</p>{% endif %} <br>{% if user_case_note_mention.is_urgent %}<p class="govuk-tag govuk-tag--red">URGENT</p>{% endif %} </td>
+						<td class="govuk-table__cell">
+							<ul class="mentions_tag">
```
Comment (kevincarrogan) [inline caseworker/templates/users/mentions.html:39]: Class names should use hyphens and not underscores.

File: caseworker/templates/users/mentions.html [hunk 5]
```diff
@@ -27,13 +27,20 @@ <h1 class="govuk-heading-l">
...
+								{% if not user_case_note_mention.is_accessed %}<li class="govuk-tag">NEW</li>{% endif %}
```
Comment (kevincarrogan) [inline caseworker/templates/users/mentions.html:40]: I wouldn't apply the tag class to the `li` itself, I would put the tag as an element inside the `li` e.g.

```
<li><span class="govuk-tag">NEW</span></li>
```

File: caseworker/templates/users/mentions.html [hunk 6]
```diff
@@ -27,13 +27,20 @@ <h1 class="govuk-heading-l">
 					<th class="govuk-table__header" scope="col"></th>
 				</tr>
 			</thead>
+			hahaha
+
```
Comment (saruniitr) [inline caseworker/templates/users/mentions.html:31]: leftover

File: caseworker/assets/styles/_mentions.scss [hunk 7]
```diff
@@ -0,0 +1,12 @@
+.mentions_tag {
+  padding: 0;
+  margin: 0;
+
+  li {
+    margin-bottom: 8px;
+    display: list-item;
+    list-style: none;
```
Comment (kevincarrogan) [inline caseworker/assets/styles/_mentions.scss:8]: `list-style: none` can be applied to the whole `ul`.

`display: list-item` is the default for an `li` so you shouldn't need to apply it again.

Comment (seijihg) [reply caseworker/assets/styles/_mentions.scss:8]: yep for some reason it was not so had to override it again.

Comment (seijihg) [reply caseworker/assets/styles/_mentions.scss:8]: yep correct it was not refresh thats why I thought it needed this :P

File: caseworker/assets/styles/_mentions.scss [hunk 8]
```diff
@@ -0,0 +1,11 @@
+.mentions-tag {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+
+  li {
+    margin-bottom: 8px;
+    text-align: center;
+    width: fit-content;
```
Comment (kevincarrogan) [inline caseworker/assets/styles/_mentions.scss:9]: I would guess that you probably don't need these anymore.

**Review by Tllew — APPROVED** [review]
Looks good to me

**Review by depsiatwal — APPROVED** [review]
lgtm

**Pull request merged**